app/assets/stylesheets/header-footer-only.scss

```diff
@@ -1,12 +1,16 @@
+$govuk-compatibility-govuktemplate: true;
+$govuk-use-legacy-palette: false;
+
+@import "govuk_publishing_components/govuk_frontend_support";
+@import "govuk_publishing_components/components/_cookie-banner";
+
```

Comment:
We seem to be pulling in the feedback component, do we need to include the css for it too? https://github.com/alphagov/static/blob/6a0d8a619e56063c81c193458bff92618f163306/app/views/root/_error_page.html.erb#L39

Reply:
Oooh, interesting, hadn't thought about that. I'll look into that, thanks.

Reply:
Feedback component sass added.